packages/requisitions/src/utils.ts

```diff
@@ -95,6 +56,18 @@ export const getNextRequestRequisitionStatus = (
   return nextStatus ?? null;
 };
 
-export const getStatusTranslation = (status: RequisitionNodeStatus) => {
-  return statusTranslation[status];
+export const getNextResponseRequisitionStatus = (
+  currentStatus: RequisitionNodeStatus
+): RequisitionNodeStatus | null => {
+  const currentStatusIdx = responseRequisitionStatuses.findIndex(
+    status => currentStatus === status
+  );
+  const nextStatus = responseRequisitionStatuses[currentStatusIdx + 1];
+  return nextStatus ?? null;
 };
+
+export const canDeleteRequisition = (
+  requisitionRow: ResponseRequisitionRowFragment | RequestRequisitionRowFragment
+): boolean =>
+  requisitionRow.status === RequisitionNodeStatus.Draft ||
+  requisitionRow.status === RequisitionNodeStatus.New;
```

Comment:
Can only delete requests, at the moment. In the future, should be able to delete adhoc/manual response reqs with no linked shipments..